internal/storage/storage.go

```diff
@@ -40,6 +40,7 @@ type Storage interface {
 	ConnectorsUninstall(ctx context.Context, id models.ConnectorID) error
 	ConnectorsGet(ctx context.Context, id models.ConnectorID) (*models.Connector, error)
 	ConnectorsList(ctx context.Context, q ListConnectorsQuery) (*bunpaginate.Cursor[models.Connector], error)
+	ConnectorsScheduleForDeletion(ctx context.Context, id models.ConnectorID) error
```

Comment:
When booting the payments application and loading plugins we're going to have to filter out the ones that have the scheduled_for_deletion flag set to true, so that we don't needlessly create workers for things that are meant to be gone, right? Were you going to handle that in this PR?

Reply:
Good catch, I forgot about that. I just pushed a new commit with the fix